src/Auth/Reducer/feedsReducer.js

```diff
@@ -67,6 +67,42 @@ export default function (state = initialState, action) {
                 allFeeds: [action.payload, ...state.allFeeds],
             });
 
+        case types.LIKE_ARTICLES_SUCCESS:
+            const { index, likedArticle } = action.payload;
+            updatedFeeds = [...state.allFeeds];
+            updatedFeeds[index] = likedArticle;
+            return Object.assign({}, state, {
+                ...state,
+                allFeeds: updatedFeeds,
+            });
+
+        case types.LIKE_GIF_SUCCESS:
+            const { indexNumber, likedGif } = action.payload;
+            updatedFeeds = [...state.allFeeds];
+            updatedFeeds[indexNumber] = likedGif;
```

Comment:
this is very wrong. you can pass the gif id, look for the index using findIndex on the feeds and then you can continue

Reply:
Ok sir. i have made the changes using findIndex()